src/js/controllers/onboarding/collectEmail.js

```diff
@@ -30,14 +30,20 @@ angular.module('copayApp.controllers').controller('collectEmailController', func
     walletId = data.stateParams.walletId;
     wallet = profileService.getWallet(walletId);
     $scope.data.accept = true;
+    walletService.getAddress(wallet,false,function(err,addr){
+      if(!err){coinAddress=addr;}else{$log.debug('Error getting address');}
```

Comment:
To make this a little cleaner, I would do this here:

```
walletService.getAddress(wallet, false, function(err, addr) {
  if (err) {
    $log.debug('Error getting address');
    return;
  }

  coinAddress=addr;
});
```

It's basically good practice to avoid using `else` unless you absolutely have to.